Feature option is not being reflected when app renders

## Summary
After disabling `application-template-wrapper`, the top-level wrapper div is still rendered and the application is still nested under it.

## Steps to setup:
1. Brand new ember application
2. `ember install @ember/optional-features`
3. `ember feature:disable application-template-wrapper`

#### My `config/optional-features.json`
```
{
  "application-template-wrapper": false
}
```

#### My `application.hbs`
```
<div class="ember-view">
  {{!-- The following component displays Ember's default welcome message. --}}
  {{welcome-page}}
  {{!-- Feel free to remove this! --}}

  {{outlet}}
</div>
```

#### Rendered in the browser:
![image](https://user-images.githubusercontent.com/1843695/36731823-e3268a4a-1b99-11e8-904d-306dd61ac811.png)

#### Ember -v
```
ember-cli: 3.1.0-beta.1
node: 8.7.0
os: darwin x64
```

#### debugging console:
```
DEBUG: -------------------------------
DEBUG: Ember      : 3.1.0-beta.2
DEBUG: Ember Data : 3.1.0-beta.1
DEBUG: jQuery     : 3.3.1
DEBUG: -------------------------------
```

Here is a link to the ember app I set up for this: https://github.com/dorilla/ember-test-no-wrap

Thanks for your help! And I'm assuming i did something wrong somewhere in my setup, any help is appreciated.

I can reproduce this. Seems like the problem is [this line](https://github.com/emberjs/ember-optional-features/blob/master/index.js#L23) doesn't do what I think. It calls [`project.require`](https://github.com/ember-cli/ember-cli/blob/master/lib/models/project.js#L346) which then calls [`resolve.sync`](https://github.com/ember-cli/ember-cli/blob/master/lib/models/project.js#L330) like this:

```
> resolve.sync('config/optional-features.json', { basedir: '/private/tmp/ember-test-no-wrap' })
Error: Cannot find module 'config/optional-features.json' from '/private/tmp/ember-test-no-wrap'
```

I suppose we could just call `require` manually (using `this.project.root`), or perhaps there is another Ember CLI API we are supposed to use here?

cc @rwjblue

--- a/index.js
+++ b/index.js
@@ -20,7 +20,7 @@ module.exports = {
     let features = {};
 
     try {
-      Object.assign(features, this.project.require('config/optional-features.json'));
+      Object.assign(features, this.project.require('./config/optional-features.json'));
     } catch(err) {
       if (err.code !== 'MODULE_NOT_FOUND') {
         throw err;

--- a/tests/optional-features-test.js
+++ b/tests/optional-features-test.js
@@ -10,7 +10,7 @@ class Project {
   require(path) {
     let features = this.features;
 
-    if (path === 'config/optional-features.json' && typeof features === 'object') {
+    if (path === './config/optional-features.json' && typeof features === 'object') {
       return this.features;
     } else {
       let error = new Error(`Invalid path: ${path}`);

